README.md

```diff
@@ -1,5 +1,7 @@
 # Mayastor Control Plane "2.0"
 
+[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B39852%2Fgit%40github.com%3Aopenebs%2Fmayastor-control-plane.git.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B39852%2Fgit%40github.com%3Aopenebs%2Fmayastor-control-plane.git?ref=badge_shield&issueType=license)
+
```

Comment:
While you're here, can you fix the build badge? I think the problem is the branch name is set to the old master name

Reply:
Done.

Reply:
Huh, but doesn't seem to work? Although if we open the link it seems to show passing.
https://mayastor-ci.mayadata.io/buildStatus/icon?job=Mayastor-Control-Plane%2Fdevelop